src/Modules/Data/Commands.Data.Test/GetPowerBIDatasetTests.cs

```diff
@@ -7,18 +13,19 @@ namespace Microsoft.PowerBI.Commands.Data
     [TestClass]
     public class GetPowerBIDatasetTests
     {
-        public static CmdletInfo InvokePowerBIRestMethodCmdletInfo { get; } = new CmdletInfo($"{GetPowerBIDataset.CmdletVerb}-{GetPowerBIDataset.CmdletName}", typeof(GetPowerBIDataset));
+        public static CmdletInfo GetPowerBIDatasetCmdletInfo { get; } = new CmdletInfo($"{GetPowerBIDataset.CmdletVerb}-{GetPowerBIDataset.CmdletName}", typeof(GetPowerBIDataset));
 
         [TestMethod]
         [TestCategory("Interactive")]
         [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
-        public void GetPowerBIDatasetTest()
+        public void EndToEndGetPowerBIDataset()
         {
             using (var ps = System.Management.Automation.PowerShell.Create())
             {
                 ProfileTestUtilities.ConnectToPowerBI(ps);
-                ps.AddCommand(InvokePowerBIRestMethodCmdletInfo).AddParameter("Scope", "Organization");
+                ps.AddCommand(GetPowerBIDatasetCmdletInfo).AddParameter("Scope", "Organization");
                 var result = ps.Invoke();
+                TestUtilities.AssertNoCmdletErrors(ps);
```

Comment:
It's probably best to use the same Arrange-Act-Assert comment arrangement as other tests to provide guidance for test coding style.